### ISSUE TITLE
AttributeError: 'StandardScaler' object has no attribute 'mean_'

### ISSUE THREAD

**Issue by Herb996** [ming024/FastSpeech2#61]
File "preprocess.py", line 15, in <module>
    preprocessor.build_from_path()
  File "/user/twx1046597/FastSpeech2-master/preprocessor/preprocessor.py", line 108, in build_from_path
    pitch_mean = pitch_scaler.mean_[0]
AttributeError: 'StandardScaler' object has no attribute 'mean_'

how to do ?

**Comment by ming024** [comment]
@Herb996 What's your sklearn version?

**Comment by psydok** [comment]
I have the same problem...
```
!pip3 list
...
scikit-image                  0.16.2        
scikit-learn                  0.23.2
...
```

**Comment by EuphoriaCelestial** [comment]
> File "preprocess.py", line 15, in
> preprocessor.build_from_path()
> File "/user/twx1046597/FastSpeech2-master/preprocessor/preprocessor.py", line 108, in build_from_path
> pitch_mean = pitch_scaler.mean_[0]
> AttributeError: 'StandardScaler' object has no attribute 'mean_'
> 
> how to do ?

I had the same error yesterday, there are 2 things you can try:
check sklearn, scikit-learn version
check TextGrid file's path. Print `tg_path` in line 77 of preprocessor.py and make sure its correct.

**Comment by ming024** [comment]
@EuphoriaCelestial thanks a lot! closed #61

**Comment by yileld** [comment]
hi，i found another situation when using biaobei dataset. In line 172 of  preprocessor.py
`wav, _ = librosa.load(wav_path)`
it didnt set the sampling_rate, so if your dataset isnt 22050kHz, it will result in return pitch becoming empty list.
